src/database/standalone-sqlite.test.ts

```diff
@@ -1044,6 +1046,44 @@ describe('StandaloneSqliteDatabase', () => {
       assert.equal(bundlesDb.prepare(sql).get({ id: fromB64Url(id0) }).cnt, 1);
     });
 
+    it('should update previous_unbundle_filter_id when unbundle_filter_id is not nul', async () => {
+      let previousUnbundleFilterId = bundlesDb
+        .prepare(sql)
+        .get({ id: fromB64Url(id1) }).previous_unbundle_filter_id;
+
+      assert.equal(previousUnbundleFilterId, null);
+
+      await db.saveBundle({
+        ...bundleId1,
+        unbundleFilter: '{"always": true}',
+      });
+
+      previousUnbundleFilterId = bundlesDb
+        .prepare(sql)
+        .get({ id: fromB64Url(id1) }).previous_unbundle_filter_id;
+
+      assert.equal(previousUnbundleFilterId, 1);
+    });
+
+    it('should update previous_index_filter_id when index_filter_id is not nul', async () => {
+      let previousIndexFilterId = bundlesDb
+        .prepare(sql)
+        .get({ id: fromB64Url(id1) }).previous_index_filter_id;
+
+      assert.equal(previousIndexFilterId, null);
+
+      await db.saveBundle({
+        ...bundleId1,
+        indexFilter: '{"always": true}',
+      });
+
+      previousIndexFilterId = bundlesDb
+        .prepare(sql)
+        .get({ id: fromB64Url(id1) }).previous_index_filter_id;
+
+      assert.equal(previousIndexFilterId, 1);
+    });
+
```

Comment:
We need a test here to check that the previous values are updated correctly when they are not null.